Doc/c-api/structures.rst

```diff
@@ -70,6 +70,17 @@ the definition of all other Python objects.
       (((PyObject*)(o))->ob_type)
 
 
+.. c:macro:: Py_IS_TYPE(o, type)
+
+   This macro is used to check whether the type of object *o* is the *type*.
```

Comment:
I suggest to remove "This macro ..." and only describe the behavior. See Py_SET_REFCNT() for example.

I also suggest to remove the "It expands to ..." part, since it's no longer a macro and I don't think that it's worth it to decribe the implementation.

Or maybe add something like: ``Equivalent to: Py_TYPE(o) == type.``